CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
 ## Enhancements
-* None
-
+* Added support for finding Realm-level permissions in Query-based Realms using `realm.getPermissions()` ([#2036]
```

Comment:
Please move `.` to end of sentence: "Added ... using `realm.getPermissions()`. ([..."